src/github.com/getlantern/flashlight/settings/settings.go

```diff
@@ -51,7 +52,10 @@ func Load(version, revisionDate, buildDate string) {
 		AutoReport: true,
 		AutoLaunch: true,
 		ProxyAll:   false,
-		InstanceID: uuid.New(),
+		// There is no true privacy or security in instance ID.  For that, we rely on
+		// transport security.  Hashing MAC would buy us nothing, since the space of
+		// MACs is trivially mapped, especially since the salt would be known
+		InstanceID: base64.StdEncoding.EncodeToString(uuid.NodeID()),
```

Comment:
It actually seems like this shouldn't be stored in settings at all since it should never change and since we don't even really want to store it to disk independently from the OS. It seems to me that we should remove InstanceID from the setting struct entirely and just have GetInstanceId be `return base64.StdEncoding.EncodeToString(uuid.NodeID())`